CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # Changelog
 
+## Unreleased
+
+### Breaking Changes
+
+- Use `sentryClientName` instead of `sdk.identifier` ([#1135](https://github.com/getsentry/sentry-dart/pull/1135))
```

Comment:
This should be under `7.0.0` section.

Reply:
Ok, doesn't matter that CI is complaining?

Reply:
No, its a feature branch yet.